etcdmain/etcd.go

```diff
@@ -520,6 +520,8 @@ func identifyDataDirOrDie(dir string) dirType {
 func setupLogging(cfg *config) {
 	capnslog.SetGlobalLogLevel(capnslog.INFO)
 	if cfg.debug {
+		formatter := capnslog.NewPrettyFormatter(os.Stdout, true)
```

Comment:
Stderr is used by capnslog's default formatter. Thought Stdout is more appropriate for log output. Not quite sure.